csv-concatenator/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8
+
+ENV MAIN_SCRIPT="concatenator.py" \
+    RABBITMQ_QUEUE=""
+
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+COPY concatenator.py extractor_info.json ./
+CMD python concatenator.py
```

Comment:
Do you need to set the CLOWDER_VERSION as environment variable?

Reply:
this extractor should work with both Clowder 1 and 2, so i don't want to hardcode a specific version here